src/platform/functions/index.js

```diff
@@ -117,12 +122,13 @@ exports.scrapeEventbrite = functions
             .firestore()
             .collection('events')
             .doc('scraped-events')
-            .collection('toronto')
+            .collection(city)
             .add(event)
             .catch((err) => functions.logger.info(err));
         })();
       });
       functions.logger.info('Uploaded to firestore');
     });
     res.send(collectiveEventsArray);
+    return functions.logger.info('Scraping Successful');
```

Comment:
```suggestion
   functions.logger.info('Scraping Successful');
```
no need to return here